shipyard-controller/handler/remoteurl_validator_test.go

```diff
@@ -0,0 +1,81 @@
+package handler
+
+import (
+	"testing"
+
+	"github.com/keptn/keptn/shipyard-controller/common"
+	common_mock "github.com/keptn/keptn/shipyard-controller/common/fake"
+)
+
+func Test_RemoteURLValidator(t *testing.T) {
+	tests := []struct {
+		url        string
```

Comment:
Also here I would suggest having a `name string` property to have the purpose of the test explicit